addons/hr_expense/models/hr_expense.py

```diff
@@ -310,9 +356,11 @@ def create_expense_from_attachments(self, attachment_ids=None, view_type='tree')
 
         for attachment in attachments:
             expense = self.env['hr.expense'].create({
-                'name': attachment.name.split('.')[0],
+                'name': 'product',
```

Comment:
Should probably be changed to something else, and be translated

Reply:
no, it was asked to name these expenses that are created when we upload attachments -  product. But indeed it is missing translation :)